Changelog.md

```diff
@@ -6,11 +6,14 @@ Language Features:
  * Yul: Introduce builtin ``blobbasefee()`` for retrieving the blob base fee of the current block.
  * Yul: Introduce builtin ``blobhash()`` for retrieving versioned hashes of blobs associated with the transaction.
  * Yul: Introduce builtin ``mcopy()`` for cheaply copying data between memory areas.
+ * Yul: Introduce builtins ``tload()`` and ``tstore()`` for transient storage access.
+
```

Comment:
I don't understand. That's the entry for this PR.

Reply:
I mean the whitespace. Almost every PR so far had to touch it and we keep changing it back and forth :) See https://github.com/ethereum/solidity/pull/14757/files#r1457682000. Or #14285.

I mean, it's a small detail and I feel bad even complaining about it, but I have to say it at least once because otherwise this madness won't stop :P

Reply:
Ah, I see it now. :D